src/bindings/python/CMakeLists.txt

```
@@ -354,11 +353,10 @@ if(ENABLE_PYTHON_PACKAGING)
     add_custom_command(OUTPUT ${openvino_telemetry_meta_info_file}
         COMMAND ${CMAKE_COMMAND} -E remove_directory
         "${telemetry_python_package_prefix}"
-        COMMAND "${Python3_EXECUTABLE}" "${OpenVINO_Telemetry_SOURCE_DIR}/setup.py"
-                    --no-user-cfg
-                    --quiet
-                    build
-                        --executable "/usr/bin/python3"
+        COMMAND "${Python3_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/wheel" -m build
```

```suggestion
        COMMAND "${Python3_EXECUTABLE}" "${OpenVINO_Telemetry_SOURCE_DIR}/wheel" -m build
```
shouldn't it be like this?

---

You're right, fixed.